nym-wallet/src-tauri/src/operations/mixnet/rewards.rs

```diff
@@ -0,0 +1,48 @@
+use crate::error::BackendError;
+use crate::nymd_client;
+use crate::state::State;
+use mixnet_contract_common::IdentityKey;
+use std::sync::Arc;
+use tokio::sync::RwLock;
+
+#[tauri::command]
+pub async fn claim_operator_reward(
+    state: tauri::State<'_, Arc<RwLock<State>>>,
+) -> Result<(), BackendError> {
+    nymd_client!(state)
+        .execute_claim_operator_reward(None)
+        .await?;
+    Ok(())
+}
+
+#[tauri::command]
+pub async fn compound_operator_reward(
+    state: tauri::State<'_, Arc<RwLock<State>>>,
+) -> Result<(), BackendError> {
+    nymd_client!(state)
+        .execute_compound_operator_reward(None)
+        .await?;
+    Ok(())
+}
+
+#[tauri::command]
+pub async fn claim_delegator_reward(
+    mix_identity: IdentityKey,
+    state: tauri::State<'_, Arc<RwLock<State>>>,
+) -> Result<(), BackendError> {
+    nymd_client!(state)
+        .execute_claim_delegator_reward(mix_identity, None)
```

Comment:
As I was saying on keybase, we really don't want to be ignoring the `fee` argument here and passing a `None` all the time, since then we are not going to be able to use the simulated gas fees (keybase://chat/nymtech#dev-core/14687)